hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/checksum/TestContainerChecksumTreeManager.java

```diff
@@ -533,4 +536,56 @@ public void testChecksumTreeFilePath() {
     assertEquals(checksumFile.getAbsolutePath(),
         ContainerChecksumTreeManager.getContainerChecksumFile(container).getAbsolutePath());
   }
+  @Test
+  void testGetContainerChecksumInfoSuccess() throws IOException {
+    String content = "mock-checksum";
+    Files.write(checksumFile.toPath(), content.getBytes());
+
+    ByteString result = checksumManager.getContainerChecksumInfo(container);
+
+    assertEquals(content, result.toStringUtf8(), "Checksum content mismatch");
+  }
+  @Test
+  void testGetContainerChecksumInfoFileNotFound() {
+
+    assertThrows(FileNotFoundException.class, () -> {
+      checksumManager.getContainerChecksumInfo(container);
+    });
+  }
+  @Test
+  void testGetContainerChecksumInfoIOException() throws IOException {
+    Files.write(checksumFile.toPath(), "test".getBytes());
+
+    // Make file unreadable
+    boolean changed = checksumFile.setReadable(false);
+
+    if (!changed && checksumFile.canRead()) {
+      System.out.println("Warning: Could not make file unreadable for test.");
+    }
+
+    assertThrows(IOException.class, () -> {
+      checksumManager.getContainerChecksumInfo(container);
+    });
+  }
+  @Test
+  void testChecksumFileExistTrue() throws IOException {
+    Files.write(checksumFile.toPath(), "dummy content".getBytes());
+
+    Container mockContainer = mock(Container.class);
+    when(mockContainer.getContainerData()).thenReturn(container);
+
+    assertTrue(ContainerChecksumTreeManager.checksumFileExist(mockContainer),
+            "Expected true since checksum file exists");
+  }
+  @Test
```

Comment:
Add a space between each test cases.
```suggestion

  @Test
```